src/util/toSubscriber.ts

```diff
@@ -1,19 +1,25 @@
 import {PartialObserver} from '../Observer';
 import {Subscriber} from '../Subscriber';
-import {$$rxSubscriber} from '../symbol/rxSubscriber';
+import {Subject} from '../Subject';
 
 export function toSubscriber<T>(
   nextOrObserver?: PartialObserver<T> | ((value: T) => void),
   error?: (error: any) => void,
   complete?: () => void): Subscriber<T> {
 
-  if (nextOrObserver && typeof nextOrObserver === 'object') {
```

Comment:
Actually, this change resulted in a circular reference issue I didn't notice until now... so I'm going to have to think about it.